### DISCUSSION TITLE
Make a weekly .NET MAUI Community Standup

### DISCUSSION THREAD

**Discussion by ncarandini** [dotnet/maui#101] [General] [answered]
It would be great if you could arrange a weekly appointment like the [ASP.NET Community Standup](https://www.youtube.com/playlist?list=PL1rZQsJPBU2StolNg0aqvQswETPcYnNKL) dedicated to Maui, where each week we can follow development of the project, hosted by @davidortinau, @jamesmontemagno et all. (like Damian Edwards with Scott Hanselman et all. on the .NET Core show).

Actually we need to follow a number of different  sources (blog pages, twitch channels, GitHub Maui project issues, ...) to stay up to date. It would be really helpful to have a single source where every week we can go and have a resumé of what's going on, what has been done and what's coming next.

Moreover, someone could act like Jon Galloway with a list of other cool sources of info (some interesting blog, code spike, twitch recording, etc.).

Please, do this!

**Comment by charlesroddie** [comment]
Would make sense once there is a community, i.e. after release.

**Reply by trampster** [reply]
.NET Manu has platforms that are listed as 'Community Supported' (like linux) if you wait until after release to build that community, then how will those community supported platforms get built?

You need to build the community in order to get a release.

**Comment by ncarandini** [comment]
> Would make sense once there is a community, i.e. after release.

@charlesroddie I don't agree, just look at the first ASP.NET vNext (even prior of naming it Core!) so you can better understand what I mean: [ASP.NET Community Standup - Sept 2nd, 2014 - Introduction to ASP.NET vNext, how and why?](https://www.youtube.com/watch?v=1rLbT6pBtak&list=PL1rZQsJPBU2StolNg0aqvQswETPcYnNKL&index=188&t=0s)

More over, the idea of open sourcing .NET MAUI from day one is to receive feedbacks and proposal, and the weekly appointment can help bring devs onboard.

> I think also an official blog like Xamarin Blog https://devblogs.microsoft.com/xamarin/
> that contains small article and all contents, link, references about all .Net MAUI update and progress.

@giuseppenovielli it's a good idea, but a different one so please open a new issue.

**Comment by GeraudFabien** [comment]
In a first time it seems intresting to have a [github review](https://www.youtube.com/watch?v=I7whBbUrMcc)
Maybe a weekly review and a monthly standup would be more useful at first since there is a need to discuss more about github issue.

**Comment by PureWeen** [comment, marked as answer]
We definitely will!!

We currently have daily meetings and once we hit a good point with the repository and everything we'll transition to have more frequent standups!!
